spec/features/researchers/coach_assignments_spec.rb

```diff
@@ -64,12 +64,11 @@
     expect(page).to have_text "Current Coach/Moderator: user2@example.com"
   end
 
-  it "doesn't allows for the assigning of no coach" do
+  it "doesn't allows for the assigning of coach if no group exists" do
```

Comment:
@ericcf I should add the unit tests as well. The feature test does not in fact cover the validation, but it tests the same logic of the validation in the conditional "link_to_if" updates.